phac-nml/irida-next
# Migrate primary IDs to UUIDs

## What does this PR do and why?
This PR updates all the primary id keys for our database tables from integers to UUIDs.

## Screenshots or screen recordings
_Screenshots are required for UI changes, and strongly recommended for all other pull requests._

## How to set up and validate locally
1. In the console while on the `main` branch, `bin/rails db:drop/create/migrate/seed`
2. Switch to this current branch `migrate-ids-to-uuids`, and run `bin/rails db:migrate`
3. Ensure the migration worked and no failures occurred.
4. Test the website functionality.
5. Ensure Tests pass locally.

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
